core/src/main/java/org/bitcoinj/base/AddressParser.java

```diff
@@ -39,6 +39,44 @@ public interface AddressParser {
      */
     Address parseAddress(String addressString, Network network) throws AddressFormatException;
 
+    /**
+     * Parse an address for any known/configured network
+     * @param addressString string representation of address
+     * @return A validated address object
+     * @throws AddressFormatException invalid address string
+     * @deprecated Use {@link #parseAddress(String)}
+     */
+    @Deprecated
```

Comment:
Do we need this deprecation? The `AddressParser` class was introduced in the current cycle, right?

Reply:
I think so, because we released the `-alpha1` binary. We can remove the deprecation after the next alpha release.

Reply:
I added a comment on the `@Deprecation` line noting that the deprecation should be removed _before_ 0.17 final.